tests/utils/test_crypt_util.py

```diff
@@ -34,6 +36,11 @@
 from tests.tools import *
 from subprocess import CalledProcessError
 
+
+def is_python_version_26():
+    return sys.version_info.major == 2 and sys.version_info.minor == 6
```

Comment:
Sigh. Python 2.6 *sucks*. The .major and .minor attributes appeared in 2.7. This ought to work:

```
return sys.version_info[0] == 2 and sys.version_info[1] == 6
```